app/src/main/java/com/example/party_player/MainActivity.kt

```diff
@@ -82,7 +80,7 @@ class MainActivity : AppCompatActivity() {
         })
     }
 
-    fun nextSong() {
+    private fun nextSong() {
```

Comment:
I changed this function to private for more clarification.

Reference: https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules

Reply:
I agree... and good citation.